<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>LoginWindow</class>
    <widget class="QMainWindow" name="LoginWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>860</width>
                <height>490</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QFrame" name="frame">
                <property name="geometry">
                    <rect>
                        <x>60</x>
                        <y>40</y>
                        <width>691</width>
                        <height>351</height>
                    </rect>
                </property>
                <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                </property>
                <widget class="QWidget" name="verticalLayoutWidget">
                    <property name="geometry">
                        <rect>
                            <x>0</x>
                            <y>0</y>
                            <width>691</width>
                            <height>351</height>
                        </rect>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QLabel" name="label">
                                        <property name="text">
                                            <string>账号：</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="Line" name="line">
                                        <property name="orientation">
                                            <enum>Qt::Vertical</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="user_name_line_edit"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="Line" name="line_5">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_2">
                                <item>
                                    <widget class="QLabel" name="label_2">
                                        <property name="text">
                                            <string>密码：</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="Line" name="line_2">
                                        <property name="orientation">
                                            <enum>Qt::Vertical</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="password_line_edit"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="Line" name="line_6">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_3">
                                <item>
                                    <widget class="QPushButton" name="sign_in_button">
                                        <property name="text">
                                            <string>登录</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="Line" name="line_3">
                                        <property name="orientation">
                                            <enum>Qt::Vertical</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="register_button">
                                        <property name="text">
                                            <string>注册</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="Line" name="line_7">
                                        <property name="orientation">
                                            <enum>Qt::Vertical</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="exit_button">
                                        <property name="text">
                                            <string>退出</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="Line" name="line_4">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </widget>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
